ci_framework/roles/repo_setup/tasks/artifacts.yml

```diff
@@ -1,5 +1,19 @@
 ---
-- name: Get DLRN hash
-  ansible.builtin.get_url:
-    url: "{{ cifwm_repo_setup_dlrn_uri }}{{ cifmw_repo_setup_os_release }}{{ cifmw_repo_setup_dist_major_version }}-{{ cifmw_repo_setup_branch }}/{{ cifmw_repo_setup_promotion }}/delorean.repo.md5"
+- name: Run repo-setup=get-hash
+  ansible.builtin.shell: |
+    {{ cifmw_repo_setup_basedir }}/venv/bin/repo-setup-get-hash \
+      --dlrn-url {{ cifwm_repo_setup_dlrn_uri }} \
+      --os-version {{ cifmw_repo_setup_os_release }}{{ cifmw_repo_setup_dist_major_version }} \
+      --tag cifmw_repo_setup_promotion }} \
```

Comment:
ah, didn't see it - missing `{{` according to molecule :)

Reply:
Done